testing/vcs/CMakeLists.txt

```diff
@@ -978,33 +978,34 @@ cdat_add_test(test_vcs_mercator_edge
 )
 
 # Rename baseline
-cdat_add_test(test_vcs_continents
-  "${PYTHON_EXECUTABLE}"
-  ${cdat_SOURCE_DIR}/testing/vcs/test_vcs_continents.py
-  ${BASELINE_DIR}/test_vcs_continents.png
-)
+#cdat_add_test(test_vcs_continents
```

Comment:
@doutriaux1 why comment out tests? The release user will see won't be running tests so I don't know a strong need for this? thoughts?